interfaces/builtin/network_control.go

```diff
@@ -204,8 +204,8 @@ capability setuid,
 # are virtual and don't show up in /dev
 /dev/net/tun rw,
 
-# Access to sysfs interfaces for tun/tap device settings.
-/sys/devices/virtual/net/tap*/** rw,
+# Access to sysfs interfaces for tun/tap/mstp/bchat device settings.
+/sys/devices/virtual/net/{tap*,mstp*,bchat*}/** rw,
```

Comment:
Hm I guess the only concern here is that those drivers do not appear to be in the mainline kernel. So perhaps this should be part of the custom-device interface?

Reply:
So I see 2 issues with that

1) The issue is that for this customer there isn't yet a "device" in /dev/ that we could associate these accesses with in a custom-device, and even if there was custom-device isn't ready yet
2) Even if this is a custom kernel driver, it is explicitly networking related (from the `net` in the sysfs path), so the logical place to put it is with network-control. If this kernel driver isn't present on the device, then these accesses aren't providing access to anything.